man/io_uring_prep_msg_ring.3

```diff
@@ -47,6 +54,17 @@ fields. The use case may be anything from simply waking up someone waiting
 on the targeted ring, or it can be used to pass messages between the two
 rings.
 
+.BR io_uring_prep_msg_ring_cqe_flags(3)
+is similar to
+.BR io_uring_prep_msg_ring(3) .
+But requests an additional parameter
+.I cqe_flags
+which is used to set the
+.I flags
+field at CQE side. That way, you can set the CQE flags field
```

Comment:
on the CQE side

Reply:
Are these addition flags or the only flags? I think that should be clarified.

Reply:
Sorry @axboe , I am not sure what you meant by this last sentence. What do you mean by "addition flags or the only flags"?

Reply:
Thanks for the clarification. I've added an extra line to clarify that `io_uring` could potentially set additional bits in this field.